code/clue.js

```diff
@@ -8,28 +8,142 @@ const mrGreen = {
   age: 45,
   image: 'assets/green.png',
   occupation: 'Entrepreneur'
-}
+};
+
+const proessorPlum = {
```

Comment:
You don't need whole this file, right? Let's delete it

Reply:
Sorry, but do not understand the above comment.🤔   Could you elaborate ?

Reply:
I meant that you have 2 `clue.js` files in your project, but this one has only default code provided by Technigo as a starter

Reply:
Ah now I understand, I removed the extra js file 😄